enterprise/kernel/src/main/java/org/neo4j/kernel/impl/enterprise/lock/forseti/SharedLock.java

```diff
@@ -214,6 +215,26 @@ public String describeWaitList()
     }
 
     @Override
+    public void collectOwners( Set<ForsetiClient> owners )
+    {
+        for ( AtomicReferenceArray<ForsetiClient> ownerArray : clientsHoldingThisLock )
```

Comment:
we can potentially go over and build that list of owners and that list of owners was never the case for this lock